otterdog/models/repository.py

```diff
@@ -463,6 +465,15 @@ def validate(self, context: ValidationContext, parent_object: Any) -> None:
                     f"while only values ['/' | '/docs'] are allowed.",
                 )
 
+        if is_set_and_valid(self.gh_pages_visibility):
```

Comment:
for preventing problems in other situations, I would add a check the plan is actually enterprise when using the property, there are some examples above in the code to check for the plan.

Reply:
agreed, I've updated the code